src/Symfony/Bundle/Resources/public/graphql-playground/index_body.css

```diff
@@ -0,0 +1,468 @@
+html {
```

Comment:
is this a vendor file ? Could you update the `update-js.sh` script accordingly?

Reply:
good catch, I didn't realize there was a script that deleted everything in this directory and recopied from node_modules. I moved the CSS file up one level so it's unaffected by the `update-js.sh` script. This seems like the correct approach since it's not actually a vendor file.